Sources/Filters/Sources/CubeSource/index.js

```diff
@@ -192,7 +192,7 @@ function vtkCubeSource(publicAPI, model) {
       .apply(points);
 
     // Define quads
-    const polys = macro.newTypedArray(model.pointType, numberOfPolys * 5);
+    const polys = macro.newTypedArray('Uint16Array', numberOfPolys * 5);
```

Comment:
no need to use the macro when the type of the array is not dynamic, simply use new `Uint16Array`